qadence_protocols/types.py

```diff
@@ -25,3 +25,17 @@ class Protocols(StrEnum):
 
 
 qadence_available_protocols = Protocols.list()
+
+
+class ReadOutOptimization(StrEnum):
+    # basic inversion and maximum likelihood estimate
+    MLE = "mle"
+
+    # constrained inverse optimization
+    CONSTRAINED = "constrained"
+
+    # matrix free measurement mitigation
+    MTHREE = "mthree"
+
+    # majority voting
+    MV = "majority_vote"
```

Comment:
```suggestion
    MAJ_VOTE = "majority_vote"
```

Or something like this.